src/mixins/overlayable.js

```diff
@@ -158,15 +158,15 @@ export default {
     },
     hideScroll () {
       if (this.$vuetify.breakpoint.mdAndDown) {
-        document.documentElement.classList.add('ovf-hidden')
+        document.documentElement.classList.add('overflow-y-hidden')
       } else {
         window.addEventListener('mousewheel', this.scrollListener)
         window.addEventListener('keydown', this.scrollListener)
       }
     },
     showScroll () {
       if (this.$vuetify.breakpoint.mdAndDown) {
-        document.documentElement.classList.remove('ovf-hidden')
+        document.documentElement.classList.remove('overflow-y-hidden')
```

Comment:
Before you had .ovf-hidden hide all overflow, is there a reason for only y here now?

Reply:
because the documentElement already has the overflow-x:hidden; attribute, in src/stylus/elements/_global.styl